dm/dm/config/security.go

```diff
@@ -72,6 +74,26 @@ func (s *Security) LoadTLSContent() error {
 	return nil
 }
 
+// DumpTLSContent dump tls certs data to file.
+func (s *Security) DumpTLSContent() error {
+	if !utils.IsFileExists(s.SSLCA) {
```

Comment:
Can users specify `ssl-ca-bytes` while left `s.ssl-ca` empty?

Reply:
yes, and dmctl use [this way](https://github.com/pingcap/ticdc/blob/b855baca0bea81e785841ccb19e615484a267abc/dm/dm/ctl/master/operate_source.go#L108) to transfer ssl-certs-data to dm-master

Reply:
Don't understand what you mention.. What will this function do if path `s.SSLCA` is invalid or empty?

Reply:
nice catch！i don't consider this case, i have two plan to slove this issue
A:  simply return err and this would cause task paused with error
B:  generate valid path for user and dump cert data to it

if you agree with plan A, no changes would be required
if you agree with plan B, i will add a test and fix in next commit

Reply:
I agree with plan B. It's hard for users to specify a right `SSLCA`, especially when sources transfer from workers to workers

Reply:
impl plan B in https://github.com/pingcap/ticdc/pull/3260/commits/ce791107df8cb1aaeea546c7a23747e082458f25, ptal 😬